x-pack/plugins/cloud/public/plugin.tsx

```diff
@@ -49,6 +60,13 @@ interface CloudStartDependencies {
   security?: SecurityPluginStart;
 }
 
+export interface CloudStart {
+  /**
+   * A React component that provides a pre-wired `React.Context` which connects components to Engagement services.
```

Comment:
`Engagement services` should probably be changed

Reply:
Nice catch.